examples/tutorial/auto_parallel/test_ci.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euxo pipefail
+
+pip install titans
+pip install pulp
+conda install coin-or-cbc
+colossalai run --nproc_per_node 4 auto_parallel_with_resnet.py -s
```

Comment:
`-s` is no longer needed if we only use synthetic dataset. This tutorial is only for user to quickly start a script without any preparation.

Reply:
Sure